examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Actually not, datadir is core part. So what do you think, should we include `cache_dirs` in `Test` API?